src/components/azure-pg/restore-db.job.ts

```diff
@@ -14,8 +14,9 @@ interface RestoreDbJobArgs {
   postRestoreScript?: string;
 }
 
-// renovate: datasource=docker depName=registry.gitlab.factory.social.gouv.fr/socialgouv/docker/azure-db versioning=2.6.1
-const SOCIALGOUV_DOCKER_AZURE_DB = "2.6.1";
+// renovate: datasource=docker depName=ghcr.io/socialgouv/docker/azure-db versioning=6.0.1
+const SOCIALGOUV_DOCKER_IMAGE = "ghcr.io/socialgouv/docker/azure-db";
+const SOCIALGOUV_DOCKER_VERSION = "6.0.1";
```

Comment:
The renovate commit should be next to the version variable 

```suggestion
const SOCIALGOUV_DOCKER_IMAGE = "ghcr.io/socialgouv/docker/azure-db";
// renovate: datasource=docker depName=ghcr.io/socialgouv/docker/azure-db versioning=6.0.1
const SOCIALGOUV_DOCKER_VERSION = "6.0.1";
```